pkgs/servers/komga/default.nix

```
@@ -0,0 +1,37 @@
+{ stdenv
+, fetchurl
+, lib
+, makeWrapper
+, jdk11_headless
+}:
+
+stdenv.mkDerivation rec {
+  pname = "komga";
+  version = "0.157.0";
+
+  src = fetchurl {
+    url = "https://github.com/gotson/${pname}/releases/download/v${version}/${pname}-${version}.jar";
+    sha256 = "sha256-PkQL61fKplt6h1jcFCIMER+ZfzowDP466dR1AaDHw5Q=";
+  };
+
+  dontUnpack = true;
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  installPhase = ''
+    mkdir -p "$out/bin"
+    mkdir -p "$out/share/lib"
+    cp -a "$src" "$out/share/lib/komga.jar"
+    makeWrapper ${jdk11_headless}/bin/java $out/bin/komga --add-flags "-jar $out/share/lib/komga.jar"
+  '';
```

We can avoid copying the jar here (h/t [`bundletool`](https://github.com/NixOS/nixpkgs/blob/365557e0431c60cba097e678fd7c0ddbb3afd9db/pkgs/development/tools/bundletool/default.nix#L18):

```suggestion
  installPhase = ''
    runHook preInstall
    makeWrapper ${jdk11_headless}/bin/java $out/bin/komga --add-flags "-jar $src"
    runHook postInstall
  '';
```

(Also, you forgot the hooks :)